payloads/v10/channel.ts

```diff
@@ -885,6 +886,10 @@ export enum EmbedType {
 	 * Link embed
 	 */
 	Link = 'link',
+	/**
+	 * Auto moderation alert embed
+	 */
+	AutoModerationMessage = 'auto_moderation_message',
```

Comment:
It seems this went undocumented: https://github.com/discord/discord-api-docs/pull/4860#pullrequestreview-977819498. Should this still be kept?

Reply:
Yeah, its still present in the object, but since embed types are considered """deprecated"""... Maybe slap an `@unstable` on it

Reply:
Maybe `@unstable This embed type is currently not documented by Discord, as it can be changed in near future. But still present in the API`?

Reply:
the hacky embed is gonna be replaced with an object